actions/arteria_delivery_service.py

```diff
@@ -45,29 +51,125 @@ def is_in_end_state(link_state):
 
         return links_results
 
+
+class ArteriaStagingQuerier(ArteriaQuerierBase):
+
+    staging_successful = 'staging_successful'
```

Comment:
Although it is pretty with the inheritance -- is there some specific reason for wanting to have "type specific" statuses? Instead of having, lets say, a general "failed", "pending", "in progress", etc. I guess the caller knows when he's speaking to the staging part and the delivery part.